extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -104,8 +104,10 @@ export class PublishOptionsDialog {
 				const row = this.optionsTable?.selectedRows![0];
 				// data[row][1] contains the option display name
 				const displayName = this.optionsTable?.data[row!][1];
+				const description = this.optionsModel.getOptionDescription(displayName);
 				await this.descriptionText?.updateProperties({
-					value: this.optionsModel.getOptionDescription(displayName)
+					value: description,
+					ariaLive: description
```

Comment:
aria-live accepts the following values: assertive, off and polite, could you please update the typings?

Reply:
updated the comment in the typings to specify the expected values. I'm surprised it was announcing the description when I was passing in the wrong string